doc/en/user/source/security/csp.rst

```diff
@@ -227,8 +283,38 @@ It does not matter whether a rule is enabled or disabled when searching precedin
 directives. The keyword ``NONE`` can be used to specify that no header value will be assigned to
 requests that match this rule.
 
+Testing
+```````
+
+The :guilabel:`Test Content Security Policy` form allows a URL to be checked, reporting the CSP
+header value that would be set for a GET request to that URL. This form will test the current CSP
+configuration in the page to allow administrators to verify the changes before saving them to the
+configuration file.
+
+Enter the URL to test in the :guilabel:`Test URL` text field and press the :guilabel:`Test` button
+to perform the test. The :guilabel:`Content-Security-Policy header value` text field will contain
+the CSP for the test URL with the string `NONE` being shown if no header would be set.
+
+.. figure:: images/csp-test.png
+
+   Test CSP with URL
+
+.. _security_csp_references:
+
+System Administrator CSP Settings
```

Comment:
This collects csp application properties and environmental variables that were previously documented in tutorials.